test/compare_inputs_and_trajectory.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+import ase.io
+import pickle
+import ase
+import multiprocessing as mp
+import os
+import sys
+from tqdm import tqdm
+import argparse
+
+def get_starting_structure_from_input_dir(input_dir):
+    metadata_path = os.path.join(input_dir, "metadata.pkl")
+    sort_path = os.path.join(input_dir, "ase-sort.dat")
+    m = open(metadata_path, "rb")
+    metadata = pickle.load(m)
+    m.close()
+    sorts = []
+    with open(sort_path, "r") as f:
+        for line in f:
+            sort, resort = line.split()
+            sorts.append(int(sort))
+    # pre-vasp sort to post-vasp structure
+    input_atoms = metadata["adsorbed_bulk_atomsobject"][sorts]
+    return input_atoms
+
+def min_diff(atoms_init, atoms_final):
+    positions = (atoms_final.positions-atoms_init.positions)
+    fractional = np.linalg.solve(atoms_init.get_cell(complete=True).T,
+                                     positions.T).T
+    for i, periodic in enumerate(atoms_init.pbc):
+        if periodic:
+            # Yes, we need to do it twice.
+            # See the scaled_positions.py test.
```

Comment:
Could you briefly explain in a comment what this method does? Also, what scaled_positions.py are you referring to?

Reply:
Added explanation for the function, deleted line 32-33 comments (forgot to delete when I moved the code over, scaled_position.py is not part of the test scripts)